package.json

```diff
@@ -246,28 +243,28 @@
     "ws": "8.2.3",
     "yargs": "16.2.0"
   },
-  "resolutions": {
```

Comment:
While pnpm does respect this field, it's confusing since it's not how you'd look up docs for this field in pnpm. Moved to `pnpm.overrides`